src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.card {
+  &:nth-child(-n + 4) {
+    margin-bottom: $gap;
+  }
+
+  @include cardStyle();
+
+  & > * {
+    display: flex;
+  }
```

Comment:
I dont think you need this. remove

Reply:
when I delete, this piece of code, the structrure of card is crumbling